src/Bladeburner/Bladeburner.ts

```diff
@@ -38,9 +46,9 @@ import { formatTime } from "../utils/helpers/formatTime";
 import { joinFaction } from "../Faction/FactionHelpers";
 import { isSleeveInfiltrateWork } from "../PersonObjects/Sleeve/Work/SleeveInfiltrateWork";
 import { isSleeveSupportWork } from "../PersonObjects/Sleeve/Work/SleeveSupportWork";
-import { WorkStats, newWorkStats } from "../Work/WorkStats";
+import { newWorkStats, WorkStats } from "../Work/WorkStats";
 import { getEnumHelper } from "../utils/EnumHelper";
-import { PartialRecord, createEnumKeyedRecord, getRecordEntries } from "../Types/Record";
+import { createEnumKeyedRecord, getRecordEntries, PartialRecord } from "../Types/Record";
```

Comment:
Did prettier do this? Why are so many import lines changing?

Reply:
Global IDE settings on my part. Want me to undo it?

Reply:
Either way is fine, but in the future you should configure your IDE to not change things you aren't explicitly touching.